python/ql/src/semmle/python/Files.qll

```diff
@@ -98,16 +98,16 @@ class Folder extends Container {
 
     /** DEPRECATED: Use `getAbsolutePath` instead. */
     deprecated override string getName() {
-        folders(this, result, _)
+        result = this.getAbsolutePath()
     }
 
     /** DEPRECATED: Use `getBaseName` instead. */
     deprecated string getSimple() {
-        folders(this, _, result)
+        result = this.getAbsolutePath()
```

Comment:
Are you sure about this? Are the second and third argument of `folders` really _both_ equal to `this.getAbsolutePath()`?

Reply:
good catch :eyes: that was some sloppy copy-paste :grimacing:

I force pushed an amended commit so this change will not be included in the final merge :+1: